test/unix/dune

```diff
@@ -8,8 +8,8 @@
   (modules test_interop_sclang)
   (libraries osc-unix test_common))
 
-(alias
-  (name   runtest)
+(rule
```

Comment:
it should be possible to convert this to a proper `test` stanza ?

Reply:
Yeah, good idea :+1: